packages/protocol/test/L1/TaikoL1TestBase.sol

```diff
@@ -310,16 +316,17 @@ abstract contract TaikoL1TestBase is TestBase {
 
     function _signAssignment(
         address signer,
-        TaikoData.ProverAssignment memory assignment,
+        AssignmentHook.ProverAssignment memory assignment,
         address taikoAddr,
         bytes32 blobHash
     )
         internal
         view
         returns (bytes memory signature)
     {
-        bytes32 digest =
-            LibProposing.hashAssignment(assignment, taikoAddr, blobHash);
+        bytes32 digest = 0;
+        // TODO
+        // AssignmentHook.hashAssignment(assignment, taikoAddr, blobHash);
```

Comment:
I cannot comment `line 383` for whatever reason (most probably because it has not changed) but it is not valid anymore:
`tko.approve(address(L1), amountTko);`

The deployed `assignmentHook` has to be approved as the `spender`.

Reply:
Tests need to changed later.